### PR TITLE
Allow sending a single file using the files param in Messageable.send

### PR DESCRIPTION
**Pull request by BeatButton**

### CODE REVIEW

File: discord/abc.py [hunk 1]
```diff
@@ -656,7 +656,7 @@ def send(self, content=None, *, tts=False, embed=None, file=None, files=None, re
         file: :class:`File`
             The file to upload.
         files: List[:class:`File`]
-            A list of files to upload. Must be a minimum of 2 and a
+            A list of files to upload. Must be a minimum of 1 and a
```
Comment (Rapptz) [inline discord/abc.py]: Remove "Must be a minimum of 1". Just make it so the maximum can be 10.

File: discord/abc.py [hunk 2]
```diff
@@ -705,8 +705,8 @@ def send(self, content=None, *, tts=False, embed=None, file=None, files=None, re
                 file.close()
 
         elif files is not None:
-            if len(files) < 2 or len(files) > 10:
-                raise InvalidArgument('files parameter must be a list of 2 to 10 elements')
+            if len(files) < 1 or len(files) > 10:
```
Comment (Rapptz) [inline discord/abc.py]: Remove the first segment of this if statement.

**Comment by BeatButton** [comment]
Changed.

**Comment by Rapptz** [comment]
Rebased.

**Pull request closed without merging**